src/Directory.Build.props

```diff
@@ -39,6 +39,8 @@
 	
 	<UNO_UWP_BUILD>true</UNO_UWP_BUILD>
 	<DefineConstants Condition="'$(UNO_UWP_BUILD)'!='true'">$(DefineConstants);HAS_UNO_WINUI</DefineConstants>
+	<DefineConstants Condition="$(UnoTargetFrameworkOverride.ToLower().StartsWith('monoandroid')) OR $(UnoTargetFrameworkOverride.ToLower().EndsWith('-android'))">$(DefineConstants);CURRENT_PLATFORM_ANDROID</DefineConstants>
```

Comment:
Does `ToLowerInvariant` work here? Could _very theoretically_ be problem in Turkey for example